docs/portal.rst

```diff
@@ -0,0 +1,208 @@
+=====================
+The Management Portal
```

Comment:
This document needs a lot of work. See comments.

There are sections with titles with no bodies. Remove them if they are not necessary.